content/en/core/overview/architecture.md

```diff
@@ -32,13 +32,70 @@ A free and open source SQL database that we use for analytics queries for displa
 
 ### cht-core
 
-[cht-core](https://github.com/medic/cht-core) is the application that most users interact with. It's an [AngularJS](https://angularjs.org) single page responsive web application.
+[cht-core](https://github.com/medic/cht-core) is composed of the following web applications:
 
-We use an offline first strategy which means the data is stored on the client and all pages can load immediately regardless of whether you have a fast connection, slow connection, or no connection at all. The data is stored in [PouchDB](https://pouchdb.com) which replicates changes back and forth in the background with the server CouchDB.
+#### CHT Web Application
+The CHT Web Application is the main entry point for must users, providing a large variety of [features](https://docs.communityhealthtoolkit.org/apps/features/).
 
-We use [Enketo](https://enketo.org) to render configured xforms and help with styling and dynamic elements such as show/hide and validation rules.
+Click [here](https://github.com/medic/cht-core/tree/master/webapp) to see the application source code.
 
-We use the [nools](https://github.com/C2FO/nools) rules engine to compute the upcoming tasks and monthly targets of the users.
+##### Technology
+The CHT Web Application is [reactive](https://angular.io/guide/rx-library), responsive and a single page application built with [Angular](https://angular.io/) and [NgRx](https://ngrx.io) frameworks.
+
+It uses an offline first strategy which means the data is stored on the client and all pages can load immediately regardless of whether the user has a fast connection, slow connection, or no connection at all. The data is stored in [PouchDB](https://pouchdb.com) which replicates changes back and forth in the background with the server CouchDB.
```

Comment:
Here I tried to follow what we had before: "It uses [tech] for [reason]"
But I think a table or bullets would look cleaner, for example:

- [Enketo](https://enketo.org): Used to render...